templates/zerver/help/allow-anyone-to-join-without-an-invitation.md

```diff
@@ -0,0 +1,49 @@
+# Allow anyone to join without an invitation
+
+By default, new users can join a Zulip organization with or without an email
+invitation.
+
+However, if you are a realm administrator that wants to limit the membership of
+your organization, you can choose to limit new members to those with email
+invitations.
+
+## Limiting new membership to users invited by organization members
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner
+of the right sidebar.
+
+2. Select **Administration** from the dropdown menu that appears.
+
+    ![Administration dropdown](/static/images/help/administration.png)
+
+3. Upon clicking **Administration**, your view will be replaced with the
+**Administration** page. You will be located on the **Organization settings**
+tab by default.
+
+    ![Administration page](/static/images/help/admin-window.png)
+
+4. In the **Organizations settings** section, locate the **E-mail invitation required**
+checkbox.
+
+    ![Email invitation required](/static/images/help/invite-required.png)
+
+    To enable limit new members of your organization to those who have been
+    invited to do so, click the **E-mail invitation required** checkbox.
```

Comment:
This doesn't sound right to me. I would either remove **enable** or change **limit** to **limiting**, the first option sounds better imo. (So "To limit new members ...")